content/login-loop-pop.md

```diff
@@ -132,6 +154,44 @@ sudo update-initramfs -c -k all
 sudo shutdown -r now
 ```
 
+### Recovery
+
+We have an article on accessing the Recovery Partition [here](https://support.system76.com/articles/pop-recovery/)
+
+To access Recovery, turn your computer off, then turn it back on and hold down the space bar immediately. In the menu that appears, select PopOS Recovery, and let it boot.
+
+Once it boots, close out of the installation window or choose “try demo mode” (be sure not to choose any install or repair options, as this could result in data loss).
+
+With the installation window closed, open a Terminal, mount the drive, and chroot (change to root) into the system (refer to the Repair and Chroot sections of the Recovery article).
+
+Once chrooted in, run the following commands:
+
+```bash
+sudo apt clean
+sudo apt update -m
+sudo dpkg --configure -a
+sudo apt install -f
+sudo apt full-upgrade
+sudo apt autoremove --purge
+```
+
+Then:
+
+```bash
+sudo apt install --reinstall plymouth gdm3 gnome-shell pop-desktop
```

Comment:
I would also reinstall the kernel and rebuild the initramfs

apt install --reinstall grub-efi-amd64 linux-generic linux-headers-generic
update-initramfs -c -k all

Reply:
Added.